src/load-service.cc

```diff
@@ -733,6 +741,16 @@ service_record * dirload_service_set::load_reload_service(const char *fullname,
             #if SUPPORT_CGROUPS
             rvalps->set_cgroup(std::move(settings.run_in_cgroup));
             #endif
+            #if SUPPORT_CAPABILITIES
+            rvalps->set_cap(std::move(cap_iab), settings.secbits.get());
+            #endif
+            if (settings.nice_is_set) rvalps->set_nice(settings.nice);
```

Comment:
To handle service reload properly, this needs to clear the `nice_is_set` value in the service (`rvalps`) if it is clear in the settings that were read.  I suggest making `set_nice` take two parameters: the "is set" flag and the value. The call here then doesn't need to be guarded by `if (settings.nice_is_set)`.

Same for `set_oom_adj`